INSTALL.md

```diff
@@ -31,6 +31,7 @@ doctrine:
                 stcollect:               CrEOF\Spatial\ORM\Query\AST\Functions\PostgreSql\STCollect
                 stsnaptogrid:            CrEOF\Spatial\ORM\Query\AST\Functions\PostgreSql\STSnapToGrid
                 stoverlaps:              CrEOF\Spatial\ORM\Query\AST\Functions\PostgreSql\STOverlaps
+		stgeomfromgeojson:       CrEOF\Spatial\ORM\Query\AST\Functions\PostgreSql\STGeomFromGeoJson
```

Comment:
@mdesloover can you check why GitHub complain ? Thanks

Reply:
j'ai pas ajouté les tests, je m'en occupe dès que possible... MErci